packages/did-method-element/src/provider.ts

```diff
@@ -19,13 +19,14 @@ type ElementNodeConfigs = {
       version: string;
     }
   ];
+  elementAnchorContract?: string;
```

Comment:
I might complain that this is actually required outside of ganache... and in 99% of test or mainnet cases.